### ISSUE TITLE
Cryptic error in gh actions when lockfile creation failed.

### ISSUE THREAD

**Issue by olivroy** [r-lib/pak#632]
I don't really know how this happened.

I get the same locally with `pak::lockfile_create()`...

My imports are 

```
Imports:
    bigD,
    commonmark
    readr,
    dplyr,
    gt,
    janitor,
    lubridate,
    parsedate,
    pkgload,
    plotly,
    quarto,
    rlang (>= 1.1.0),
    rvest,
    stringr,
    tidyr,
    usethis,
    yaml
```

in DESCRIPTION 

```r
  ✖ Creating lockfile '.github/pkg.lock' [759ms]
  
  
  Error: 
  ! error in pak subprocess
  Caused by error in `if (any(bad <- types == "")) { …`:
  ! missing value where TRUE/FALSE needed
  ---
  Backtrace:
  1. pak::lockfile_create(c(deps, extra_deps), lockfile = ".github/pkg.lock", …
  2. pak:::remote(function(...) { …
  3. err$throw(res$error)
  ---
  Subprocess backtrace:
   1. base::withCallingHandlers(cli_message = function(msg) { …
   2. get("lockfile_create_internal", asNamespace("pak"))(...)
   3. prop$solve()
   4. private$plan$solve(policy = private$policy)
   5. pkgdepends:::pkgplan_solve(self, private, match.arg(policy))
   6. self$resolve()
   7. pkgdepends:::pkgplan_resolve(self, private)
   8. pkgdepends:::synchronise(self$async_resolve())
   9. asNamespace("pkgcache")$synchronise(...)
  10. base::stop(priv$value)
  11. global (function (e) …
  Execution halted
```

**Comment by gaborcsardi** [comment]
A comma is missing after `commonmark`.

**Comment by olivroy** [comment]
Ooops, thanks!  Should have known..

 Could pak validate the DESCRIPTION ? AFAICT, this is not really implemented anywhere. `desc::desc_validate()` is not implemented. and I ran `usethis::use_tidy_description()`, but it didn't let me know either.

**Comment by olivroy** [comment]
Hi @gaborcsardi, just wanted to report back.

It seems that my fix at r-lib/pkgdepends#398 worked. I tried using dev pak to confirm:

Here is the new error message I get:

```r
Error:  1  deps for  1/1  pkgs [⠼] Done                                                
! error in pak subprocess
Caused by error in `get_remote_types(refs)`:
! Cannot parse package: commonmark readr.
ℹ See `?pkgdepends::pkg_refs()` for supported package sources.
````

Clearly an improvement in case you wanted to add this to NEWS!
